obsplus/events/utils.py

```diff
@@ -243,7 +243,10 @@ def _bump_version(version):
 
 
 def make_origins(
-    events: catalog_or_event, inventory: obspy.Inventory, depth: float = 1.0
+    events: catalog_or_event,
+    inventory: obspy.Inventory,
+    depth: float = 1.0,
+    phases: Optional[Iterable] = None,
```

Comment:
We might make the default ("P", "p"). The tuple will be immutable so you can use it as the default and avoid the none check.